### PR TITLE
hardcode processing info

### LINKED ISSUE #65: Add package version information to Processing Information Modals
https://github.com/AlexsLemonade/refinebio-frontend/issues/22 Introduced the Processing information Modals.

We need to add version information on these modal dialogs for each sample.

image

This information should be exposed from the backend AlexsLemonade/refinebio#229

### PR DESCRIPTION
**Pull request by ramenhog**
## Issue Number
Closes #65 

## Purpose/Implementation Notes
* Hardcode package version information based on technology

## Types of changes

What types of changes does your code introduce?
New feature (non-breaking change which adds functionality)

## Functional tests
Each supported technology shows the appropriate copy and version info.

### Salmon & tximport
screen shot 2018-06-29 at 10 07 18 am
### Affymetrix SCAN
screen shot 2018-06-29 at 10 08 25 am
### Illumina SCAN
screen shot 2018-06-29 at 10 51 18 am
### Submitter processed
screen shot 2018-06-29 at 10 14 18 am

### CODE REVIEW

File: src/containers/Experiment/SamplesTable.js [hunk 1]
```diff
@@ -372,55 +381,67 @@ function ScanModal({ sample, scanType }) {
...
       </table>
     </div>
   );
 }
 
+function _getScanVersionInfo(scanType) {
+  if (scanType === PIPELINES.AffymetrixSCAN) {
+    return {
+      'SCAN.UPC': '2.20.0',
+      Brainarray: '22.0.0'
```
Comment (dongbohu) [inline src/containers/Experiment/SamplesTable.js]: Should be `'Brainarray'`, right?

Comment (dongbohu) [reply src/containers/Experiment/SamplesTable.js]: Just realized that it is permitted in JavaScript (but illegal in Python). Let's quote `Brainarray` just for consistency.

Comment (ramenhog) [reply src/containers/Experiment/SamplesTable.js]: Oops, actually it's more consistent if I remove all the quotes, so I'll do that.

Comment (ramenhog) [reply src/containers/Experiment/SamplesTable.js]: Oh, nevermind. Prettier actually formatted those objects automatically, so we should probably go with that.

**Pull request merged**